### PR TITLE
Modelcard

### LINKED ISSUE #11: Add model card
## Issue Title
Add model cards to the repository

## Issue Description
I would like to add model cards to the repository following the instructions provided in the [Hugging Face Hub model card template](https://github.com/huggingface/huggingface_hub/blob/main/src/huggingface_hub/templates/modelcard_template.md) or the original [Mitchell paper](https://arxiv.org/abs/1810.03993).

## Additional Information
Please let me know if there are any additional steps I need to take to add model cards to the repository.

### PR DESCRIPTION
**Pull request by richardbartels**
Fixes #11

### CODE REVIEW

**Comment by richardbartels** [comment]
Fixes #11

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.2] - 2023-11-29
+
+### Added
+- Added an model card in the root
```
Comment (rubenpeters91) [inline CHANGELOG.md:13]: a model card

**Comment by richardbartels** [comment]
> Maybe we should put both the model card and dataset card in the docs folder? Otherwise we might get too many files in the root folder

@rubenpeters91 dat is prima wat mij betreft. Misschien wel goed om dan een referentie in de README.md op te nemen? Default voor model card is de root maar `docs/` is ook logisch

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [1.0.2] - 2023-11-29
 
 ### Added
-- Added an model card in the root
+- Added a model card in the root
```
Comment (rubenpeters91) [inline CHANGELOG.md:13]: Strictly speaking no longer in the root and maybe add a ### Changed category with moved the dataset card

Comment (rubenpeters91) [reply CHANGELOG.md]: I was too quick

**Review by rubenpeters91 — APPROVED** [review]
You probably need to solve the merge conflict in CHANGELOG, but looks good otherwise

**Pull request merged**